cachito/workers/config.py

```diff
@@ -44,6 +44,7 @@ class Config(object):
     cachito_nexus_timeout = 60
     cachito_nexus_username = "cachito"
     cachito_request_lifetime = 1
+    cachito_strict_gomod_vendor = False
```

Comment:
Could you please rename this to `cachito_gomod_strict_vendor`. I just try to keep configuration options relating to a package manager in the format of `cachito_<package manager>_<configuration option>`.